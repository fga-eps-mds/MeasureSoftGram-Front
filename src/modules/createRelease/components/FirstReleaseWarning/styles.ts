import styled from 'styled-components';

export const Header = styled.div`
  margin-bottom: 64px;
  width: 600px;

  & > h1 {
    font-size: 42px;
    font-weight: 400;
  }

  & > p {
    font-size: 16px;
  }
`;

export const Body = styled.div`
  width: 600px;
  & > div:first-child {
    display: flex;

    & > p {
      margin: 40px;
    }
  }
`;
