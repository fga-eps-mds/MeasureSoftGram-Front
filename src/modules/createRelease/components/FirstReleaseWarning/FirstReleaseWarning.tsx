import React from 'react';
import * as Styles from './styles';

export default function FirstReleaseWarning() {
  return (
    <>
      <Styles.Header>
        <h1>Planejar Release</h1>
      </Styles.Header>
      <Styles.Body>
        <p>
          Por ser a primeira release do seu projeto, será disponibilizada a seguir uma pré-configuração base gerada pelo
          sistema.
        </p>{' '}
        <br />
        <p>
          Caso deseje, existe a opção de fazer alterações em cada tela, utilizando o botão correspondente para ativar ou
          desativar a edição daquela configuração específica. Dessa forma, é possível personalizar a pré-configuração de
          acordo com suas necessidades.
        </p>
      </Styles.Body>
    </>
  );
}
