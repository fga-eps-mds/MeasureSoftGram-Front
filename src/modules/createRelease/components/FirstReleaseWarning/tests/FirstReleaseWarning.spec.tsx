import React from 'react';
import { render } from '@testing-library/react';
import FirstReleaseWarning from '../FirstReleaseWarning';

describe('<FirstReleaseWarning />', () => {
  it('Deve corresponder ao snapshot', () => {
    const tree = render(<FirstReleaseWarning />);

    expect(tree).toMatchSnapshot();
  });
});
