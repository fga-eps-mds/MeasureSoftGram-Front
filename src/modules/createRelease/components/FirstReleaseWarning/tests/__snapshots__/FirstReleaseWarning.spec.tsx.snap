// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FirstReleaseWarning /> Deve corresponder ao snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styles__Header-sc-1eodkkx-0 eTclVL"
      >
        <h1>
          Planejar Release
        </h1>
      </div>
      <div
        class="styles__Body-sc-1eodkkx-1 fVtKFn"
      >
        <p>
          Por ser a primeira release do seu projeto, será disponibilizada a seguir uma pré-configuração base gerada pelo sistema.
        </p>
         
        <p>
          Caso deseje, existe a opção de fazer alterações em cada tela, utilizando o botão correspondente para ativar ou desativar a edição daquela configuração específica. Dessa forma, é possível personalizar a pré-configuração de acordo com suas necessidades.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styles__Header-sc-1eodkkx-0 eTclVL"
    >
      <h1>
        Planejar Release
      </h1>
    </div>
    <div
      class="styles__Body-sc-1eodkkx-1 fVtKFn"
    >
      <p>
        Por ser a primeira release do seu projeto, será disponibilizada a seguir uma pré-configuração base gerada pelo sistema.
      </p>
       
      <p>
        Caso deseje, existe a opção de fazer alterações em cada tela, utilizando o botão correspondente para ativar ou desativar a edição daquela configuração específica. Dessa forma, é possível personalizar a pré-configuração de acordo com suas necessidades.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
